### DISCUSSION TITLE
Nuking connected repos and changing e-mail

### DISCUSSION THREAD

**Discussion by VeArnold** [codemagic-ci-cd/codemagic-docs#2081] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

-

### Expected results

A few things:

a) is there a convenient way to nuke all connected repos - instead of going at them one-by-one?

b) this blog post still refers to Slack community support, which does not seem to exist anymore: https://blog.codemagic.io/join-our-slack-community-for-continuous-integration/

c) can I somehow update my e-mail? I log in to Codemagic using GitHub, but the linked e-mail (shown under user preferences) is different to the one that is attached to the GitHub account... I saw a thread about not being able to do so, however I am not even using this e-mail to sign in...

### Actual results

-

**Comment by mikhail-tokarev** [comment]
hey hey :D 

I'm afraid you need to delete apps one-by-one. However, you can delete the whole account and create a new one if it's ok

**Comment by mikhail-tokarev** [comment]
the article was deleted - thanks for bringing this up

**Comment by VeArnold** [comment]
Alright, I guess I will have to do the manual deleting or wait 14 days it seems :))) I guess I'll manage, thanks anyhow.

**Reply by mikhail-tokarev** [reply]
I can enforce acc deletion, no need to wait for 14 days. I just need to know team id

**Reply by VeArnold** [reply]
Oh that would be nice!
5ec2466b0476746f03539c94

Thanks

**Reply by mikhail-tokarev** [reply]
done, the account will be deleted over the night
